services/ui-src/src/components/tables/SortableNaaarStandardsTable.tsx

```diff
@@ -34,10 +34,15 @@ export const SortableNaaarStandardsTable = ({
         ? `standard_analysisMethodsUtilized${standardId}`
         : "";
 
+      // check if there is "other" text for provider type
       const coreProviderType = entity[
-        "standard_coreProviderTypeCoveredByStandard-otherText"
+        `${standard_coreProviderTypeCoveredByStandard[0].key}-otherText`
```

Comment:
this logic is a good candidate for the [compareText util Bangbay has in review](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/12079). That said, I know this is a bug fix so maybe we don't want to delay it.